runtime/lua/vim/lsp/util.lua

```diff
@@ -303,6 +310,55 @@ function M.make_floating_popup_options(width, height, opts)
   }
 end
 
+function M.jump_to_location(location)
+  if location.uri == nil then return end
+  local bufnr = vim.uri_to_bufnr(location.uri)
+  -- TODO(ashkan) this function was removed because I didn't fully comprehend
+  -- the tagstack and it was a holdover from the previous LSP efforts. If
+  -- someone wants to work on this and enable it, they can find it in the
+  -- commit d5aaad14ecdd2047089e1a018e97af1f790b3e42 in the file
+  -- `runtime/lua/vim/lsp/buf.lua` at line 103.
+  -- update_tagstack()
```

Comment:
1-line comment (+ 12-char SHA) is enough (the longer explanation could be in the commit message):

```suggestion
  -- TODO(ashkan): see d5aaad14ecdd for impl
  -- update_tagstack()
```

Reply:
I think is correct that this should _not_ use the tagstack. For those that want to to use tag behavior with LSP, we should provide a 'tagfunc' implementation. But using a function like `update_tagstack()` to emulate tagstack behavior is an anti-pattern to start width.  So I think we can just delete this entire comment (or perhaps write "a 'tagfunc' implementation would be a good complement").